rotkehlchen/accounting/mixins/event.py

```diff
@@ -79,8 +79,11 @@ def process(
     def serialize(self) -> dict[str, Any]:
         ...
 
+    def _serialize_for_debugging(self) -> dict[str, Any]:
+        return self.serialize()
+
     def serialize_for_debug_import(self) -> dict[str, Any]:
-        data = self.serialize()
+        data = self._serialize_for_debugging()
```

Comment:
I feel this is a bad name because it does reference to where it is used instead of what it does but @LefterisJP can also chime in here and give a third opinion

Reply:
Yes, makes sense, I agree. Renamed it to `serialize_all_info`